core/os/main_loop.cpp

```diff
@@ -33,11 +33,6 @@
 #include "core/script_language.h"
 
 void MainLoop::_bind_methods() {
-	ClassDB::bind_method(D_METHOD("init"), &MainLoop::init);
-	ClassDB::bind_method(D_METHOD("iteration", "delta"), &MainLoop::iteration);
-	ClassDB::bind_method(D_METHOD("idle", "delta"), &MainLoop::idle);
-	ClassDB::bind_method(D_METHOD("finish"), &MainLoop::finish);
```

Comment:
That's not a "Docs:" commit, that's changing the scripting API.

This likely requires a dedicated PR not presented as a docs PR, the doc impact of this change is secondary.

Reply:
okay I will export the second commit into a new PR in a second

Reply:
done